doc/api/worker_threads.md

```diff
@@ -156,6 +209,9 @@ Disables further sending of messages on either side of the connection.
 This method can be called when no further communication will happen over this
 `MessagePort`.
 
+The [`'close'` event][] will be emitted on both `MessagePort` instances that
```

Comment:
```suggestion
Both `MessagePort` instances that are part of the channel will emit a [`'close'` event][].
```

Reply:
I have to admit that active voice sounds a bit weird to me when referring to events :)